docs/naisjob/README.md

```diff
@@ -27,60 +27,64 @@ spec:
   schedule: "*/1 * * * *"
 ```
 
-!!! warning
-    If your Naisjob is [running in GCP](../clusters/gcp/), or if it is using [Vault sidecar](reference/#vaultsidecar) or [Securelogs](reference/#securelogs).
-    You need to shut down all containers running in the pod for the `Job` to complete.
-    See [Shutting down extra containers](#shutting-down-extra-containers) for more information.
-
 ## Re-run Naisjob
 If you don't want to run your job at a schedule, but still want to re-run your Naisjob, you either have to delete the old Naisjob first, og run under a different name.
 
 ## Applying your Naisjob to Kubernetes
 You can deploy your Naisjob just as you would deploy your Application using [NAIS deploy](../deployment/README.md).
 
 ## Shutting down extra containers
-If your Naisjob has any of these traits:
+Due to the nature of NAIS, your Naisjob might end up having a few sidecars.
+Since a sidecar typically has little to no logic to look at sibling containers, these would normally not turn off once your app completes. 
+As a result, the Naisjob would continue running indefinitely unless some entity from above sorts it out.
+To remediate this, an operator named [Ginuudan](https://github.com/nais/ginuudan) has been created to run in every cluster and observe every Naisjob.
```

Comment:
Skulle vi ha nevnt noe om at du kan skru på dette for cronjobs/jobs ved å legge til en annotasjon?

Reply:
man kan ikke skru på dette. man har ikke et valg gitt at man har deploya en naisjob etter.. midten av forrige uke

Reply:
Hvis du lager din egen `cronjob`/`job` så kan du skru det av og på. Var det jeg mente.

Reply:
da syns jeg de burde bruke naisjob :p pluss, dokumentasjonen denne siden handler om er naisjobs